src/mds/MDSRank.cc

```diff
@@ -3906,6 +3908,7 @@ void MDSRankDispatcher::handle_conf_change(const ConfigProxy& conf, const std::s
     sessionmap.handle_conf_change(changed);
     server->handle_conf_change(changed);
     mdcache->handle_conf_change(changed, *mdsmap);
+    mdlog->handle_conf_change(changed);
```

Comment:
This is also part of the minor log segment branch. Maybe just rebase on that?

Reply:
ACK. Will remove the commit from this PR.